db/views/flat_filters_v01.sql

```diff
@@ -0,0 +1,40 @@
+SELECT
```

Comment:
what about adding a comment at the top of this file to explain why we have this view? 😅